CMakeLists.txt

```diff
@@ -166,9 +164,13 @@ if(NOT WIN32 AND NOT APPLE)
   find_program(KOKKOSTOOLS_DTRACE_EXECUTABLE dtrace)
   if(KOKKOSTOOLS_DTRACE_EXECUTABLE)
     add_subdirectory(profiling/systemtap-connector)
+    set(KOKKOSTOOLS_HAS_SYSTEMTAP ON)
   else()
     message(STATUS "Skipping systemtap-connector (dtrace executable wasn't found)")
+    set(KOKKOSTOOLS_HAS_SYSTEMTAP OFF)
   endif()
+else()
+  set(KOKKOSTOOLS_HAS_SYSTEMTAP OFF)
```

Comment:
Why do you set to OFF?  I expect `#cmakedefine` will do the right thing if the variable is not defined at all.

Reply:
I preferred to be explicit. I can change it if you feel strongly about it.

Reply:
I feel like it increases complexity but can live with it